pkg/controllers/project/resolvers.go

```diff
@@ -95,10 +95,11 @@ func (h *handler) resolveProjectRegistrationNamespaceData(namespace, name string
 }
 
 func (h *handler) resolveProjectRegistrationNamespaceRoleBinding(namespace, name string, rb *rbacv1.RoleBinding) ([]relatedresource.Key, error) {
-	isProjectRegistrationNamespace, err := h.projectGetter.IsProjectRegistrationNamespace(namespace)
+	namespaceObj, err := h.namespaceCache.Get(namespace)
 	if err != nil {
```

Comment:
it's fine to return an error in a resolver since it registers itself as a handler under the hood, on an error to getting the namespace from the cache it will simply requeue itself with exponential backoff